nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/scheduling/TestStandardProcessScheduler.java

```diff
@@ -169,6 +169,7 @@ public void after() throws Exception {
      * run. This unit test is intended to verify that we have this resolved.
      */
     @Test
+    @Ignore("This test appears to be buggy")
```

Comment:
@markap14 any chance you can take a look here?  I noticed 40+MB of test output docs on surefire output loaded with exception stacks and it all related to a private static reporting task in the test classes (see below for where i changed from private to public).  However, after doing that and seeing the logs appear much better the test for ensuring reporting task stops was now failing.  I cant quite make sense of it.

Reply:
@joewitt thanks. I created NIFI-5686 and submitted a PR for htis.